src/main/java/graphql/schema/idl/errors/BaseError.java

```diff
@@ -2,15 +2,15 @@
 
 import graphql.ErrorType;
 import graphql.GraphQLError;
-import graphql.GraphQLException;
+import graphql.GraphQLInstanceException;
 import graphql.GraphqlErrorHelper;
 import graphql.language.Node;
 import graphql.language.SourceLocation;
 
 import java.util.Collections;
 import java.util.List;
 
-class BaseError extends GraphQLException implements GraphQLError {
+class BaseError extends GraphQLInstanceException implements GraphQLError {
```

Comment:
This is where I would like to have the "schema exceptions" instead - for this class of exception around building and validating schema